### PR TITLE
Frontend tooltips and removals

### PR DESCRIPTION
**Pull request by BenCampbellNelson**
Addition of tooltips to each widget, providing users with quick, on-hover descriptions for each data type (e.g., temperature, pressure, CO2). 

Also removal of humidity and webcam as unused.

### CODE REVIEW

**Comment by SamanthaCra** [comment]
Tooltips don't work when clicked on :(

**Comment by BenCampbellNelson** [comment]
> Tooltips don't work when clicked on :(

Were working on hover. Have made changes for working onClick now too. As well as a size increase!

File: fullstack/src/app/page.js [hunk 1]
```diff
@@ -11,12 +10,10 @@ export default function Home() {
       <div className="widgets">
         <Widget name="Temperature" dataKey="temperature" GraphComponent={null} />
         <Widget name="Air Pressure" dataKey="pressure" GraphComponent={null} />
-        <Widget name="Humidity" dataKey="humidity" GraphComponent={null} />
         <Widget name="Wind" dataKey="wind" GraphComponent={null} />
```
Comment (SamanthaCra) [inline fullstack/src/app/page.js:13]: Could we please update this to remove wind direction too?

File: fullstack/src/components/ui/tooltip.jsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+const Tooltip = TooltipPrimitive.Root
+
+const TooltipTrigger = TooltipPrimitive.Trigger
+
+const TooltipContent = React.forwardRef(({ className, sideOffset = 4, ...props }, ref) => (
+  <TooltipPrimitive.Content
+    ref={ref}
+    sideOffset={sideOffset}
+    className={cn(
```
Comment (SamanthaCra) [inline fullstack/src/components/ui/tooltip.jsx:18]: Can we format this so it's a bit nicer to read? Maybe run it through Prettier?

**Review by SamanthaCra — CHANGES_REQUESTED** [review]
The tooltips are looking really good. Thanks!

File: fullstack/src/components/ui/tooltip.jsx (hunk 2 above)
Comment (BenCampbellNelson) [reply fullstack/src/components/ui/tooltip.jsx:18]: Yep it's all just a prebuilt component so hadn't even looked at it!

**Review by SamanthaCra — APPROVED** [review]
Thanks for the fixes!

**Pull request merged**